controllers/ext_client.go

```diff
@@ -261,6 +261,16 @@ func createExtClient(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var extclient models.ExtClient
+	var CustomExtClient models.CustomExtClient
+	
+	err := json.NewDecoder(r.Body).Decode(&CustomExtClient);
+
+	if err != nil {
+		logger.Log(1, "error creating CustomExtClient"+err.Error())
```

Comment:
log message suggests an error occurred, but it's more of an absence of a clientid, would either log that no clientid was provided and one will be created, or just check if err == nil { extclient.ClientID = CustomExtClient.ClientID } (no log)

Reply:
@0xdcarns ok you're right i have made the changes